roles/pulp_services/README.md

```diff
@@ -0,0 +1,67 @@
+pulp_services
+=============
+
+A role to install & configure Pulp's 1st-party services
```

Comment:
I think we need to have a section defining what is 1st or 3rd party and adding a link.
Something like: if you want install ___ consider `pulp_all_services`

Reply:
I added a "Related Roles" section, which half addresses your comment.

I wonder if "1st-party" vs "3rd-party" is the correct term.

I am stickler for correct terminology, but still, this seems to be a symptom of that.

@melcorr Can you advise?

Reply:
Interesting. I have checked IBM style and they don't have any advice, so we're on our own. What IBM does advise is spelling out words under 10, so it would be first-party services, third-party services. Would internal and external services work? What grates with me is the first-party services. Third-party services seems completely fine. Internal and third-party services? WDYT? I do agree with @fao89 that we should define these terms because without a definition, we're at the mercy of the reader to invent their meaning :)

Reply:
@melcorr We discussed this at the meeting and could not identify good replacement terminology.

Even though it was more work, I added the definitions.

Reply:
Also, I changed it to "first-party" and "third-party" per your suggestion.